readme.md

```diff
@@ -230,9 +230,11 @@
 - [Preact](https://github.com/ooade/awesome-preact) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources)
 - [Next.js](https://github.com/unicodeveloper/awesome-nextjs) - Framework for server-rendered React apps.
+- [R-Shiny](https://github.com/grabear/awesome-rshiny) - Web Framework built with R.
```

Comment:
As per rules in https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md#requirements-for-your-pull-request you should add the entry in the end of the category, so you need to add it after `Hyperapp`

Reply:
also, you should add `#readme` in the end of the link and make sure it is linked proprly

Reply:
No need to specify the section name in your PR title, please rename it from `Added R-Shiny to the Front-End Development secion.` to `Add R-Shiny`

Reply:
please also rebase your PR as it has conflicts currently